retreat-playbook.md

```diff
@@ -0,0 +1,481 @@
+
+
+#### Starting a Worker Co-op
+
+This is a broad overview of the steps required to start a worker co-op that should be useful to inform the roadmapping activity. Some recommendations have also been included.
+
+| ACTIVITY | RECOMMENDATIONS AND COMMENTS |
+|----------|----------|
+| **PHASE I - DEVELOPING THE IDEA** | |
+| 1. Assemble a Group of Interested People | :heavy_check_mark: |
+| 2. Conduct a Pre-feasibility Study | :heavy_check_mark: We have project ideas on the go |
+| **PHASE II - CO-ORDINATING PRE CO-OPERATIVE ACTIVITIES** |
+| 3. Hold an organizing meeting | :palm_tree: Retreat meetings on Dec 4 & 8 :palm_tree: |
+| 4. Conduct a viability study  | Draft business plan and conduct feasibility study (including approach to Y1 financing) |
+| **PHASE III - ORGANIZING AND GETTING STARTED** || 
+| 5. Organize the association <br> Set up adhoc committies such as: <br> - Planning <br> - Tools <br> - Drafting by-laws | Draft bylaws and recommendation to submit articles of incorporation <br> Early identified issues: <br>- Nature of Board (all, subset?) <br> - Nature of Membership (tiers, flat?) <br> - Nature of Capital (shares, buy-in?) <br> - Nature of Profit (how to manage surplus; see FSCO Guide) |
+| 6. Organize the enterprise <br> - Plan the operations <br> - Plan start up financing <br> - Recruit and train staff <br> - Ensure legal compliance | Consult a lawyer over by-laws <br> Prepare member handbook: <br> - Connecting by-laws and our working processes through our values <br> - Plain-language duties and checklists for required roles (president, secretary, finance person) <br> - What it means to be a good member <br> - Archive of agreements and minutes <br> - Guide to our ways of working | 
+| 7. Hold the initial General Meeting | By-laws must be adopted within 18 months |
+
+*NB: Based on [Industry Canada's The Seven Steps in Forming a Co-operative](http://www.ic.gc.ca/eic/site/106.nsf/eng/h_00073.html#seven)*  
+*NB: Details on [FSCO Legal Requirements](http://www.fsco.gov.on.ca/en/coops/Documents/legal_reqs.pdf)*  
+*NB: For more details on resources see dcwalk's hackpad ["Summary of Legal Requirements"](https://hackmd.io/s/SJwTqa767)*
+
+## Day 1 - Visioning
+
+*Length: 3 hours*  
+*Format: Video call*
+
+| Time          | Length | Description                                                                |
+|---------------|--------|----------------------------------------------------------------------------|
+| 00:00 - 00:15 | 15mins | [Introductions](#introductions) |
+| 00:15 - 01:00 | 45mins | [5-minute sharebacks of solo visioning and readings](#5-minute-sharebacks) |
+| 01:00 - 01:15 | 15mins | Break!                                                                     |
+| 01:15 - 02:30 | 75mins | [Co-drafting vision & mission template](#Co-drafting-vision-amp-mission)   |
+| 02:30 - 03:00 | 30mins | [Finding value alignment](#Finding-value-alignment)                        |
+
+### Introductions
```

Comment:
```suggestion
### Introductions

*Facilitator: Dawn*  
*Scribe: Ben (ensure how we work is documented)*

- How we got here
- What the retreat is for and our aims for today
- Logistics
    - Recording this meeting
    - [Shared notepad](https://hackmd.io/PbNVbPUhSTKL7oEtnrzpuw)
    - Making decisions will be through full consensus of those present
- How we will work together, proposed group agreements:
    - Shared responsibility for conversation (step up listening, step up speaking)
    - One person speaking at a time
    - Honest respect with each other
    - Focused, on topic discussion
    - Keep each other on time
    - Device-free
```

Reply:
@benhylau -- here are my proposed group agreements taken from previous facilitation work! Updated in the online playbook!

Reply:
Thanks!